content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -51,5 +61,389 @@ QR-code` option to *EMV Merchant-Presented QR-code*.
+- :guilabel:`MPF Manulife Account`: Account number if applicable.
+
+Manage contracts
+----------------
+
+Once the new employee has been created, click the :guilabel:`Contracts` smart button on the
+employee record, or navigate to :menuselection:`Employees app--> Employees --> Contracts`.
+
+.. note::
+   Only **one** contract can be active simultaneously per employee, but an employee can be assigned
+   consecutive contracts during their employment.
+
+The following are critical for setting up a contract:
+
+- :guilabel:`Working Schedule`: Set as HK Standard 40 hours/week (from employee record)
+- :guilabel:`Salary Structue Type`: Set as CAP57: Hong Kong Employee.
+- :guilabel:`Work Entry Source`: Select either :guilabel:`Working Schedule`, :guilabel:`Attendances` or :guilabel:`Planning`. 
+  This field determines how the work entries are accounted for in the payslip. 
+
+   - :guilabel:`Working Schedule`: The work entries are generated automatically based on the employee's working schedule.
+   - :guilabel:`Attendances`: The work entries are generated based on the check-in/-out period logged on the *Attendances* app.
+   - :guilabel:`Planning`: The work entries are generated from planning shifts only.
+
+Under the **Salary Information** tab:
+
+- :guilabel:`Wage Type`: Select :guilabel:`Fixed Wage` for Full-time or Part-time employees, or :guilabel:`Hourly Wage` for employees who are paid hourly.
+- :guilabel:`Wage`: Monthly or Hourly depending on the company.
+- :guilabel:`Internet Subscription`: This is an **optional** field to provide additional internet allowance on top of the current salary package.
+
+  .. important::
+     Timesheets do not impact work entries in Odoo.
+
+Once all information has been setup, set the contract status to `Running` by clicking the :guilabel:`Running` button 
+in the top-right of the page.
+
+.. image:: hong_kong/hk-contract.png
+   :alt: Hong Kong employment contract
+
+.. _hong_kong/running_payslips:
+
+Generate payslips
+-----------------
+
+Once the employees and their contracts are configured, payslips can be generated in the *Payroll* app.
+
+Odoo provides **four** different salary structures under CAP57 Regulation:
+
+#. :guilabel:`Employees Monthly Pay`: To process the monthly employee salary.
+#. :guilabel:`Payment in Lieu of Notice`: To process final payment upon contract termination.
+#. :guilabel:`Long Service Payment`: Applicable to employees with more than 5 years of service upon
+   contract termination.
+#. :guilabel:`Severance Payment`: Applicable to employees with more than 2 years of service upon
+   contract termination.
+
+Before running the payslips, the accounts used in the salary rule can be adjusted by navigating to
+:menuselection:`Payroll --> Configurations --> Rules`.
+
+.. image:: hong_kong/hk-salary-rules.png
+   :alt: Hong Kong Salary Rules
+
+Odoo can create pay runs in **two** ways: via **individual** or via **batch** payslips.
```

Comment:
```suggestion
Odoo can create pay runs in **two** ways: via **individual** or **batch** payslips.
```

I recommend to have the order of the below sections match this sentence. Either switch Individual and Batches sectoins below, or swap the words in this sentence, for consistency.